app/services/braintree_service/escrow_release_helper.rb

```diff
@@ -40,18 +43,11 @@ def release_from_escrow_after_next_batch(community_id, transaction_id)
       self.delay(:run_at => next_escrow_release_time, :priority => 6).release_from_escrow_by_community_id(community_id, transaction_id)
     end
 
-    # Give a date and get back time of the given date when batch is run
-    def settlement_batch_time_per_date(date)
-      offset = ActiveSupport::TimeZone.new("Central Time (US & Canada)").formatted_offset
-      Time.new(date.year, date.month, date.day, 17, 0, 0, offset)
-    end
-
     # Give a date and get back time of next batch time
     def next_settlement_batch_time(now=Time.now)
-      todays_batch = settlement_batch_time_per_date(now)
-      tomorrows_batch = settlement_batch_time_per_date(now.tomorrow)
-
-      now < todays_batch ? todays_batch : tomorrows_batch
+      now = now.in_time_zone(BATCH_TIMEZONE)
+      now += 1.day if now.hour >= BATCH_HOUR # we've missed today's batch
+      now.change({ hour: BATCH_HOUR, min: 0, sec: 0 })
```

Comment:
I rewrote this piece a little. Decided to leave min and sec though to make it explicit for future generations.